bonk/serializers.py

```diff
@@ -140,7 +150,7 @@ def get_by_ip(cls, vrf, ip, reql=False):
     @classmethod
     def filter_by_block(cls, block, reql=False):
         return cls.filter(lambda b:
-            r.ip_prefix_contains(
+                r.ip_prefix_contains(
                 r.ip_prefix(block['network'], block['length']),
```

Comment:
These should be indented more too, right?

Reply:
yes! however I had no intention of changing this, must've slipped, fell and landed on a<tab>:wq

true story!